packages/plugin-core/src/views/utils.ts

```diff
@@ -144,7 +136,7 @@ export class WebViewUtils {
       // and hand it out to any other functions that need to use it.
       acquireVsCodeApi: `const vscode = acquireVsCodeApi(); window.vscode = vscode;`,
       themeMap: themeMap as WebViewThemeMap,
-      initialTheme: initialTheme ?? defaultInitialTheme,
```

Comment:
Maybe we should rename that parameter to `themeOverride` to make it clear that this is only intended to fixate the theme to either light, dark or custom.